### DISCUSSION TITLE
How do i consume SP api?

### DISCUSSION THREAD

**Discussion by scerelli** [amzn/selling-partner-api-models#3360] [General]
Hi everyone, 
i open a discussion here because i can't understand what should I do to consume the selling partner api. The docs is not quite clear about the process of creating a new application.

I have to develop a public fulfillment integration with shopify via shopify app, so: sync inventory via api from amazon to shopify, forward orders from shopify to amazon for fulfillment etc

How do i create a dev apps without a selling center account?

**Comment by scerelli** [comment]
So reading a little bit more the docs i found:

```
To register as a public developer (for all public applications)

    Sign into Seller Central using the credentials that you want to associate with your developer account.

    From the Partner Network menu, choose Develop Apps.

    If you have not yet completed a developer profile for this selling account, select Proceed to Developer Profile. Otherwise, select the View profile link.

    Complete the form. From the Data Access section dropdown box, choose Public Developer: I build and offer publicly available applications that are used by other companies”.
```

i don't have a selling center account and to make one it requires me to pay 39 dollars, put tons of data and register myself as a seller, is this right?

**Reply by vaidas-lungis** [reply]
> i don't have a selling center account and to make one it requires me to pay 39 dollars, put tons of data and register myself as a seller, is this right?

Yes and wait for approval that could take up a month or two 😓

**Reply by scerelli** [reply]
looks like a joke to me
